src/libs/WorkflowUtils.ts

```diff
@@ -117,9 +117,23 @@ function convertPolicyEmployeesToApprovalWorkflows({employees, defaultApprover,
             return -1;
         }
 
+        if (b.isDefault) {
+            return 1;
+        }
+
         return (a.approvers[0]?.displayName ?? '-1').localeCompare(b.approvers[0]?.displayName ?? '-1');
     });
 
+    // Add a default workflow if one doesn't exist (no employees submit to the default approver)
+    const firstWorkflow = sortedApprovalWorkflows.at(0);
```

Comment:
NAB:

```suggestion
    const firstWorkflow = sortedApprovalWorkflows[0];
```

Reply:
There is an ongoing [issue](https://github.com/Expensify/App/issues/43055#issuecomment-2197395087) related to this. In future we plan to disallow `[0]`. That's why I'll leave it as is 🙇‍♂️